src/apps/vpn/platforms/ios/iostunnel.swift

```diff
@@ -13,12 +14,36 @@ class PacketTunnelProvider: NEPacketTunnelProvider {
         }
     }()
 
+    override init() {
+        super.init()
+
+        Logger.configureGlobal(tagged: "NET", withFilePath: FileManager.logFileURL?.path)
+
+        // Copied from https://github.com/mozilla/glean/blob/c501555ad63051a4d5813957c67ae783afef1996/glean-core/ios/Glean/Utils/Utils.swift#L90
+        let paths = FileManager.default.urls(for: .applicationSupportDirectory, in: .userDomainMask)
+        let documentsDirectory = paths[0]
+        // We just can't use "glean_data". Otherwise we will crash with the main Glean storage.
+        let gleanDataPath = documentsDirectory.appendingPathComponent("glean_netext_data")
+
+        let defaults = UserDefaults(suiteName: Constants.appGroupIdentifier)
+        let telemetryEnabled = defaults!.bool(forKey: Constants.UserDefaultKeys.telemetryEnabled)
+        let appChannel = defaults!.string(forKey: Constants.UserDefaultKeys.appChannel)
+
+        Glean.shared.registerPings(GleanMetrics.Pings.self)
+        Glean.shared.initialize(
+            uploadEnabled: telemetryEnabled,
+            configuration: Configuration.init(
+                channel: appChannel != nil ? appChannel : "unknown",
+                dataPath: gleanDataPath.relativePath
+            ),
+            buildInfo: GleanMetrics.GleanBuild.info
+        )
+    }
```

Comment:
Note: Add a shutdown call to `deinit` once the new Glean release comes out. It's supposed to come out tomorrow.